go.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/shirou/gopsutil v3.21.11+incompatible
 	github.com/status-im/keycard-go v0.2.0
 	github.com/stretchr/testify v1.8.1
-	github.com/syndtr/goleveldb v1.0.1-0.20220721030215-126854af5e6d
+	github.com/syndtr/goleveldb v1.0.1
```

Comment:
just change to following?
github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7

so not need to replace later
and no need to modify accounts/abi/bind/bind_test.go

I think if path  changed, like github.com/**tendermint**/tendermint => github.com/**bnb-chain**/tendermint
we need replace it later
for goleveldb, there is no need.

Reply:
can't do as above
because pyrsm depend on v1.0.1-0.20220721030215-126854af5e6d